ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-GA-stage.yaml

```
@@ -59,6 +59,36 @@ tests:
     test:
     - ref: addon-tests-rhods
     workflow: addons-and-operators
+- as: smoke-osd-gcp-rhods-addon-baseline-fips
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      ADDONS_OPERATORS_RUN_IN_PARALLEL: "false"
+      CLUSTER_NAME_PREFIX: rhods
+      CLUSTER1_CONFIG: platform=gcp-osd;region=us-east-2;version=4.13;channel-group=stable;compute-machine-type=g4dn.2xlarge;timeout=90m;replicas=2;fips=true;ocm-env=stage
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "operator-install", "failure_type": "all", "classification": "Provisioning - Operators", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-506"},
+          {"step": "addon-install", "failure_type": "all", "classification": "Provisioning - Addons", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-506"},
+          {"step": "addon-tests-*", "failure_type": "all", "classification": "Addons tests", "jira_project": "RHODS", "jira_epic": "RHODS-10718"},
+          {"step": "cluster-install-*", "failure_type": "all", "classification":  "Cluster Creation - OSD-GCP", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-506", "group": {"name": "cluster-provision", "priority": 1}},
+          {"step": "cluster-uninstall-*", "failure_type": "all", "classification":  "Cluster Creation - OSD-GCP", "jira_project": "MSINTEROP", "jira_epic": "MSINTEROP-506", "group": {"name": "cluster-provision", "priority": 2}}
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: MSINTEROP
+      FIREWATCH_JIRA_API_TOKEN_PATH: /tmp/secrets/jira/access_token_msi
+      OPERATOR1_CONFIG: name=gpu-operator-certified;namespace=nvidia-gpu-operator;source=certified-operators;target-namespaces=nvidia-gpu-operator,
+      OPERATOR2_CONFIG: name=servicemeshoperator
+      OPERATOR3_CONFIG: name=serverless-operator;namespace=openshift-serverless
+      OPERATOR4_CONFIG: name=openshift-pipelines-operator-rh;namespace=openshift-pipelines;channel=latest
+      OPERATOR5_CONFIG: name=nfd;namespace=openshift-nfd;target-namespaces=openshift-nfd,
+      PULL_SECRET_NAME: pull-secret
+      S3_BUCKET_NAME: openshift-cli-installer
+      S3_BUCKET_PATH: openshift-ci
+    test:
+    - ref: addon-tests-rhods
+    workflow: addons-and-operators
 - as: smoke-osd-rhods-addon-baseline-fips
```

@rnetser @myakove now we'll have two OSD jobs, on AWS and OSD, so I would sugget to update the existing job name to reflect platform also, WDYT?
```suggestion
- as: smoke-osd-aws-rhods-addon-baseline-fips
```

---

agreed; I think we'll need to specify the cloud provider on all existing OSD jobs and OCP/IPI to be onboarded as well

---

@chetna14manku please update